src/openrct2-ui/windows/EditorObjectSelection.cpp

```diff
@@ -108,10 +108,12 @@ static constexpr const ObjectPageDesc ObjectSelectionPages[] = {
     { STR_OBJECT_SELECTION_WATER,                       SPR_TAB_WATER,              false },
 
     // Currently hidden until new save format arrives:
-    // { STR_OBJECT_SELECTION_TERRAIN_SURFACES,            SPR_G2_TAB_LAND,            false },
-    // { STR_OBJECT_SELECTION_TERRAIN_EDGES,               SPR_G2_TAB_LAND,            false },
-    // { STR_OBJECT_SELECTION_STATIONS,                    SPR_TAB_PARK,               false },
+    // { STR_OBJECT_SELECTION_TERRAIN_SURFACES,            SPR_G2_TAB_LAND,            true  },
+    // { STR_OBJECT_SELECTION_TERRAIN_EDGES,               SPR_G2_TAB_LAND,            true  },
+    // { STR_OBJECT_SELECTION_STATIONS,                    SPR_TAB_PARK,               true  },
     // { STR_OBJECT_SELECTION_MUSIC,                       SPR_TAB_MUSIC_0,            false },
+    // { STR_OBJECT_SELECTION_FOOTPATH_SURFACES,           SPR_TAB_SCENERY_PATHS,      false },
+    // { STR_OBJECT_SELECTION_FOOTPATH_RAILINGS,           SPR_G2_PATH_RAILINGS_TAB,   false },
```

Comment:
Are these changes relevant to be cherry picked, given they're all comments?

Reply:
Yes, they are commented out because you can’t use them until the NSF is there. But changing them now will make the diff easier to read once the NSF is there, which is why I included them.